modules/core/src/test/java/org/apache/ignite/internal/commandline/CommandHandlerParsingTest.java

```diff
@@ -690,4 +694,67 @@ private boolean requireArgs(@Nullable CommandList cmd) {
             cmd == CommandList.DATA_CENTER_REPLICATION ||
             cmd == CommandList.MANAGEMENT;
     }
+
+    /**
+     * test parsing management arguments
+     */
+    @Test
+    public void testManagementArguments() throws IOException {
+        ConnectionAndSslParameters args;
+
+        parseArgs(asList("--management", "status"));
+        parseArgs(asList("--management", "help"));
+        
+        args = parseArgs(asList("--management", "on"));
+
+        ManagementArguments arg = ((ManagementCommands)args.command()).arg();
+
+        assertTrue(arg.isEnable());
+
+        args = parseArgs(asList("--management", "off"));
+
+        arg = ((ManagementCommands)args.command()).arg();
+
+        assertFalse(arg.isEnable());
+
+        assertParseArgsThrows("Expected server URIs", "--management", "uri");
+
+        args = parseArgs(asList("--management", "uri", "http://localhost:3000"));
+
+        arg = ((ManagementCommands)args.command()).arg();
+        
+        assertTrue(arg.isEnable());
+        assertEquals(singletonList("http://localhost:3000"), arg.getServerUris());
+
+        assertParseArgsThrows("Invalid uri arguments", "--management", "uri", "http://localhost", "--wrong");
+
+        String keyStorePath = U.resolveIgnitePath("/modules/core/src/test/resources/server.jks").getAbsolutePath();
```

Comment:
Alternatively, from com.google.common.io.Resources
```java
getResource("server.jks");
```

Reply:
I don't want add dependency from guava

Reply:
It is there since GG-27090